Aptoide/aptoide-client-v8
# [feature/ASV-1735] Deeplink Navigation

**What does this PR do?**

   This PR passes parameters from the deeplink intent to be used on the wallet install dialog.

NOTE: Accept ASV-1734 first.

**Database changed?**

   No

**Where should the reviewer start?**

- [x] WalletInstallConfiguration.kt

**How should this be manually tested?**

  Check if when the intent is sent, the info is being correctly retrieved in WalletInstallManager through WalletInstallConfiguration. For example, for the following intent:

adb shell am start -a "android.intent.action.VIEW" "market://details?id=com.appcoins.wallet\&utm_source=myappcoins\&app_source=com.legacyofmobile.com"

Check if we have the correct wallet package and correct app package. You'll probably have to debug to know if this is correct.

**What are the relevant tickets?**

  [ASV-1735](https://aptoide.atlassian.net/browse/ASV-1735)


**Code Review Checklist**
